src/Hotel.java

```diff
@@ -5,24 +5,29 @@
     private String name;
     private ArrayList<Room> rooms;
 
-    Hotel(String name, ArrayList<Room> rooms) {
+    Hotel(String name, ArrayList<Room> rooms)
+    {
         this.name = name;
         this.rooms = rooms;
     }
 
-    String getName(){
+    String getName()
```

Comment:
If you don't add access modifiers to your methods, classes or properties they get package-private access level by default. That means you have access to them only within the same package. 
Always explicitly add acces modifiers like public/private/protected.